src/main/java/yahav/photomosaic/PhotoMosaicApplication.java

```diff
@@ -60,6 +58,14 @@ public void start(Stage primaryStage) throws Exception {
                 color = Color.color(red, green, blue);
                 pixelWriter.setColor(x,y,color);
             }
+        File[] file = new File("/Users/taliayahav/PhotoMosaic/src/main/resources/flower").listFiles();
+        for(File img : file){
+            Image fileImage = new Image(img.toURI().toString());
+            ImageView fileImageView = new ImageView();
+            fileImageView.setImage(fileImage);
+        }
```

Comment:
Unable to find good method to find min of each image

Reply:
What do you mean by "min of each image"?